go/libraries/doltcore/table/pipeline/procfunc_help.go

```diff
@@ -87,10 +87,6 @@ type SinkFunc func(row.Row, ReadableMap) error
 // processing, stop conditions, and error handling.
 func ProcFuncForSinkFunc(sinkFunc SinkFunc) OutFunc {
 	return func(p *Pipeline, ch <-chan RowWithProps, badRowChan chan<- *TransformRowFailure) {
-		defer func() {
-			badRowChan <- &NoTransformRowFailure
-		}()
```

Comment:
moved out of this specific OutFunc and now closing the badRowChan when any outFunc finishes.